src/main.cpp

```diff
@@ -40,7 +40,11 @@
 
 #include "binding.h"
 
+#ifdef(__WINDOWS__)
+#include "resource.h"
+#else
 #include "icon.png.xxd"
+#endif
```

Comment:
No need for the else I think? Or do you get compiler warnings if icon.png is included on windows?